docs/en/operations/settings/settings.md

```diff
@@ -998,6 +998,99 @@ The results of the compilation are saved in the build directory in the form of .
 
 If the value is true, integers appear in quotes when using JSON\* Int64 and UInt64 formats (for compatibility with most JavaScript implementations); otherwise, integers are output without the quotes.
 
+## output\_format\_json\_quote\_denormals {#settings-output_format_json_quote_denormals}
+
+Enables `+nan`, `-nan`, `+inf`, `-inf` outputs in [JSON](../../interfaces/formats.md#json) output format.
+
+Possible values:
+
+-   0 — Disabled.
+-   1 — Enabled.
+
+Default value: 0.
+
+For example, when this setting disabled, the query returns null values in output:
```

Comment:
```suggestion
**Example**

Consider the following table:

```
Here is the contents of the test_table.
Maybe we can find a more interesing name for the table?
```

When `output_format_json_quote_denormals = 0`, the query returns null values in output:
```